src/Algebra/Morphism/ToSetoid.agda

```diff
@@ -0,0 +1,49 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Basic definitions for maps to a setoid
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+
+open import Function using (_∘_)
+open import Level using (_⊔_)
+open import Relation.Binary using (Rel; _⇒_; Reflexive; Symmetric;
+                                   Transitive; IsEquivalence; Setoid)
+open import Relation.Binary.PropositionalEquality using (_≡_; refl)
+
+
+module Algebra.Morphism.ToSetoid {α β β=} (A : Set α) (S : Setoid β β=) where
```

Comment:
I don't see you using any of the contents of this file except for `_≈∘_` anywhere in this PR, and in the context `_≈∘_` is used the existing `_≗_` can be used just fine.

Therefore could you remove this file from the PR and you can always add it back in a later PR if you need it?

Reply:
IsHomo-rev is proved for arbitrary Magmas, in particular, with  _≈∘_.
Instead you suggest to include in the library only its special case for PropositionalEquality.
This looks like a poor design. Because users will program various generalizations for this lemma instead of using standard library. Agda is such a language in which methods tend to be programmed in a general way.

Reply:
> Instead you suggest to include in the library only its special case for PropositionalEquality.

Yes you're right, I was getting slightly confused but my point stands for `IsHomo-Rev`. If you use `IsInverse` in `Function.Structures` which works over arbitrary equalities then you no longer need `_≈∘_`.